evm/src/main/java/org/hyperledger/besu/evm/operation/SarOperation.java

```diff
@@ -44,7 +44,7 @@ public SarOperation(final GasCalculator gasCalculator) {
     } else {
       final int shiftAmountInt = shiftAmount.toInt();
 
-      if (shiftAmountInt >= 256) {
+      if (shiftAmountInt >= 256 || shiftAmountInt < 0) {
```

Comment:
what happens if it's == 0

Reply:
also would be great to have a test since this caused an issue

Reply:
The bytes handler handles that efficiently.

Reply:
👍